frontend/src/main/components/Users/UsersTable.js

```diff
@@ -53,20 +69,21 @@ export default function UsersTable({ users}) {
         {
             Header: 'Driver',
             id: 'driver',
-            accessor: (row, _rowIndex) => String(row.driver) // hack needed for boolean values to show up
+            accessor: (row, _rowIndex) => String(row.driver) 
         }
     ];
 
     const buttonColumn = [
+        // shows all the text columns, and then the button columns
         ...columns,
         ButtonColumn("toggle-admin", "primary", toggleAdminCallback, "UsersTable"),
-    ]
-
-    //const columnsToDisplay = showButtons ? buttonColumn : columns;
+        ButtonColumn("toggle-driver", "primary", toggleDriverCallback, "UsersTable"),
```

Comment:
Mutation testing fails in UsersTable for line 79 and 80. Looks like this wasn't caught before b/c stryker enable was used instead of stryker restore.